### ISSUE TITLE
Add social links below the hero section and footer

### ISSUE THREAD

**Issue by chrisrabe** [YanLi-FE/redot-landing-page#12]
# Description

Add a section below the hero container and footer with quick navigation icons to social links:

- Github: https://github.com/Redot-Engine/redot-engine
- Discord: https://discord.gg/redot
- Twitter/X: https://x.com/redotengine
- Reddit: https://www.reddit.com/r/RedotGameEngineMain/
- Youtube: https://www.youtube.com/@ReDotEngine

# Figma design:

URL: https://www.figma.com/design/47xbUZkEOEkDg7OBEkL5H9/ReDot-Website?node-id=2612-1499&t=HOzJzxzAfGFSYn8g-1

Hero section:
![Image](https://github.com/user-attachments/assets/69929db1-a994-4431-bcda-4aa24f6f2123)

Footer:
![Image](https://github.com/user-attachments/assets/01dac43e-6f24-4721-a18b-4150d393a534)

**Comment by opdev1004** [comment]
This one I think we should ask the design guy and others and see what they think. It's good idea but I am not sure about separating it from the logo and download button.

**Comment by chrisrabe** [comment]
Yeah this is in the designs. We currently don't have that as current state
